bob-common/src/mapper.rs

```diff
@@ -121,37 +121,38 @@ impl Virtual {
         self.vdisks.get(&id).expect("vdisk not found").nodes()
     }
 
-    pub fn get_support_nodes(&self, key: BobKey, count: usize) -> Vec<&Node> {
+    pub fn get_support_nodes(&self, key: BobKey, count: usize) -> (Vec<&Node>, Vec<&Node>) {
```

Comment:
I don't understand this change at all. Please explain the idea behind these changes. And also revert it because it's wrong anyway